### PR TITLE
Cycle ASG instances in a sustainable way

### PR DESCRIPTION
**Pull request by arunasank**
Fixes https://github.com/hotosm/tasking-manager/issues/1456

This PR:
1. Cycles ASG instances when a change is made to the LaunchConfiguration
2. Maintains 100% healthy instances

cc/ @dakotabenjamin

### CODE REVIEW

**Comment by arunasank** [comment]
I tested this on a new stack, and it works:

Screen Shot 2019-04-10 at 1 13 20 PM

File: cloudformation/tasking-manager.template.js [hunk 1]
```diff
@@ -118,11 +124,14 @@ const Resources = {
...
           'echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf',
```
Comment (arunasank) [inline cloudformation/tasking-manager.template.js]: @dakotabenjamin i know you really like the install all deps at once format, but it makes diffs to the script difficult to follow. Would you mind too much if we moved more to one command per line?

**Comment by arunasank** [comment]
Don't worry about the tests failing, because this is a merge with the `cloudformation` branch. There's a comment there about fixing the failing test: https://github.com/hotosm/tasking-manager/pull/1332#commitcomment-32804493

Thanks for flagging @ramyaragupathy 🍏

File: cloudformation/tasking-manager.template.js (hunk 1 above)
Comment (dakotabenjamin) [reply cloudformation/tasking-manager.template.js]: Agreed

**Pull request merged**